arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -74,3 +76,32 @@ def rule_gfm_1_1_3(
             uri=uri,
             modelObject=elt,
         )
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_1_7(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.1.7] Attribute xml:base must not appear in any filing document.
+    """
+    baseElements = []
+    for rootElt in (val.modelXbrl.ixdsHtmlElements):
+            for uncast_elt, depth in etreeIterWithDepth(rootElt):
+                elt = cast(Any, uncast_elt)
+                if elt.get(xmlBaseIdentifier) is not None:
+                    baseElements.append(elt)
+                if elt.tag == xhtmlBaseIdentifier:
+                    baseElements.append(elt)
```

Comment:
I understand that we're checking this because it's essentially the inline version of the XML attribute, but it might be worth a comment about why we're checking for the XHTML element when the rule only mentions the XML attribute.